src/views/AgmExtension.vue

```diff
@@ -288,7 +289,7 @@ export default class AgmExtension extends Mixins(CommonMixin, DateMixin, FilingM
     this.data.incorporationDate = this.getFoundingDate
 
     // Pre-populate the certified block with the logged in user's name (if not staff)
-    if (!this.isRoleStaff && this.getUserInfo) {
+    if (!this.IsAuthorized(AuthorizedActions.THIRD_PARTY_CERTIFY_STMT) && this.getUserInfo) {
```

Comment:
In Create UI and Edit UI, I used BLANK_CERTIFY_STATE for this.

PS Update the comments from "if staff" to "is authorized" (or whatever -- your choice).